app/src/main/java/com/khinthirisoe/bakingapp/ui/baking/view/BakingAdapter.kt

```diff
@@ -0,0 +1,81 @@
+package com.khinthirisoe.bakingapp.ui.baking.view
+
+import android.content.Context
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.khinthirisoe.bakingapp.R
+import com.khinthirisoe.bakingapp.data.model.Recipe
+import com.khinthirisoe.bakingapp.ui.base.BaseViewHolder
+import com.khinthirisoe.bakingapp.ui.base.inflate
+import kotlinx.android.synthetic.main.list_baking.view.*
+
+class BakingAdapter(
+    private val context: Context,
+    private var recipes: MutableList<Recipe>?,
+    private val clickListener: BakingRecyclerViewClickListener
+) :
+    RecyclerView.Adapter<BakingAdapter.ViewHolder>() {
```

Comment:
@KhinThiriSoe extending and implementing should not be in the new line. move it up to 17